cognee/infrastructure/llm/tokenizer/Gemini/adapter.py

```
@@ -28,12 +29,11 @@ def __init__(
             get_llm_config,
         )
 
-        config = get_embedding_config()
         llm_config = get_llm_config()
 
-        import google.generativeai as genai
+        from google import genai
```

Pylint (3.3.8)

[error] No name 'genai' in module 'google'

(E0611)